sdcm/cluster_k8s/__init__.py

```diff
@@ -1025,10 +1025,34 @@ def deploy_ingress_controller(self, pool_name: str = None):
         if not self.params.get('reuse_cluster'):
             pool_name = pool_name or self.AUXILIARY_POOL_NAME
             comma_separated_tags = ", ".join([f'{k}={v}' for k, v in self.tags.items()])
+
+            if self.params.get("cluster_backend") == "k8s-eks":
+                cpu_limit = 2
+                memory_limit = 4
+                cpu_request = 1
+                memory_request = 2
+
+                cpu_limit = convert_cpu_units_to_k8s_value(cpu_limit)
+                memory_limit = convert_memory_units_to_k8s_value(memory_limit)
+                cpu_request = convert_cpu_units_to_k8s_value(cpu_request)
+                memory_request = convert_memory_units_to_k8s_value(memory_request)
```

Comment:
JFYI:
If we set `request` and `limit` value as the same then we get `dedicated` resources and better performance.
If we set `request` and `limit` to different values like in this change then we get `context switch` situation and it worse from perf point of view.

Considering that one more `defaul` node gets created I would expect to have the first variant here - with better perf.

Reply:
And again this is how the cloud team configuration is, and I wanted that our default would be identical.

Reply:
I don't mind. Let it be so.